api/controllers/CampaignBotController.js

```diff
@@ -1,548 +1,454 @@
 "use strict";
 
+/**
+ * Imports.
+ */
 const logger = rootRequire('lib/logger');
 const mobilecommons = rootRequire('lib/mobilecommons');
 const phoenix = rootRequire('lib/phoenix')();
 const helpers = rootRequire('lib/helpers');
-
 const connOps = rootRequire('config/connectionOperations');
 const dbRbSubmissions = require('../models/campaign/ReportbackSubmission')(connOps);
 const dbSignups = require('../models/campaign/Signup')(connOps);
 const dbUsers = require('../models/User')(connOps);
 
+/**
+ * Setup.
+ */
 const CMD_REPORTBACK = (process.env.GAMBIT_CMD_REPORTBACK || 'P');
 
 /**
- * CampaignBotController
- * @constructor
- * @param {integer} campaignId - our DS Campaign ID
+ * CampaignBotController.
  */
-function CampaignBotController(campaignId) {
-  this.campaignId = campaignId;
-  this.campaign = app.getConfig(app.ConfigName.CAMPAIGNS, campaignId);
-  if (!this.campaign) {
-    return;
-  }
-
-  // @todo Config variable for default CampaignBot ID
-  // @todo Store campaignbot property on our config.campaigns to override bot
-  // on a per DS Campaign basis.
-  this.bot = app.getConfig(app.ConfigName.CAMPAIGNBOTS, 41);
+class CampaignBotController {
 
-  var mobileCommonsCampaign = this.campaign.staging_mobilecommons_campaign;
+  constructor(req, res) {
```

Comment:
@sergii-tkachenko @deadlybutter This is related to [whether to define our current User on our `CampaignBotController` or in Express request `req`]( https://github.com/DoSomething/gambit/pull/628#pullrequestreview-1236088). Any opinion on which is cleaner -- creating`this.req` and `this.res` properties on the `CampaignBotController` vs passing them to every function? 

e.g.  `this.loadUserAndSignup()` instead of `this.loadUserAndSignup(req, res)`